kubejs/server_scripts/unify.js

```diff
@@ -1,125 +1,125 @@
-                output: {
-                    "slurry": "mekanism:clean_" + name,
-                    "amount": 1
-                },
-                type: 'mekanism:washing'
-            });
-            console.log("Removing dissolution recipes for #forge:ores/" + name + " to mekanism:dirty_" + name);
-            e.remove({
-                input: '#forge:ores/' + name,
-                output: {
-                    "slurry": "mekanism:dirty_" + name,
-                    "amount": 1000
-                },
-                type: 'mekanism:dissolution'
-            });
-*/
+            /* Not working prior to KubeJS kubejs-1604.3.4.137-forge.jar, unnecessary currently.
+                        console.log(`Removing injecting recipes for mekanism:crystals/${name} to mekanism:shard_${name}`);
+                        e.remove({
+                            input: `#mekanism:crystals/${name}`,
+                            output: `mekanism:shard_${name}`,
+                            type: 'mekanism:injecting'
+                        });
+                        console.log(`Removing crystallizing recipes for mekanism:clean_${name} to mekanism:shard_${name}`);
+                        e.remove({
+                            input: {
+                                "slurry": `mekanism:clean_${name}`,
+                                "amount": 200
+                            },
+                            output: `mekanism:crystal_${name}`,
+                            type: 'mekanism:crystallizing'
+                        });
+                        console.log(`Removing washing recipes for mekanism:dirty_${name} to mekanism:clean_${name}`);
+                        e.remove({
+                            input: {
+                                "slurry": `mekanism:dirty_${name}`,
+                                "amount": 1
+                            },
+                            output: {
+                                "slurry": `mekanism:clean_${name}`,
+                                "amount": 1
+                            },
+                            type: 'mekanism:washing'
+                        });
+                        console.log(`Removing dissolution recipes for #forge:ores/${name} to mekanism:dirty_${name}`);
+                        e.remove({
+                            input: `#forge:ores/${name}`,
+                            output: {
+                                "slurry": `mekanism:dirty_${name}`,
+                                "amount": 1000
+                            },
+                            type: 'mekanism:dissolution'
+                        });
+            */
             //Uncomment and use this to see if your criteria will match a recipe, and if the recipe is what you want them to match. Useful for updating script, do not leave uncommented for pack release.
             //e.forEachRecipe({}, recipe => console.info('Recipe matched! ' + recipe));
-        };
+        }
```

Comment:
Just mentioning it here, but it applies to the entire file: please use semicolons. They prevent an entire kind of bug, and your editor can probably just insert them automatically. I know prettier does, at least.